charts/operators-installer/Chart.yaml

```diff
@@ -16,7 +16,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 3.0.2
+version: 3.0.3
```

Comment:
@garethahealy since this is a feature add please bump to 3.1.0

Reply:
I bumped to 3.1.0